docs/source/contributing_examples/example.py

```diff
@@ -182,6 +193,10 @@ def to_jams(self):
         # -- see the documentation for `jams_utils.jams_converter for all fields
 
 
+# -- this decorator allows this function to take a string or an open bytes file as input
+# -- and in either case converts it to an open file handle.
+# -- It also checks if the file exists
+# -- and, if None is passed, None will be returned 
 @io.coerce_to_bytes_io
 def load_audio(fhandle):
     """Load a Example audio file.
```

Comment:
Cool! Thas was definitely needed.

Should we also include here an example of the usage of `typing`? e.g:
```
def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
```
or 
```
def beats(self) -> Optional[annotations.BeatData]:
```
We use it a lot in the loaders but not in the example files.

Reply:
Yes good point! adding it